cli/script/command-parser.ts

```diff
@@ -43,18 +43,18 @@ function accessKeyAdd(commandName: string, yargs: yargs.Argv): void {
     isValidCommand = true;
     yargs.usage(USAGE_PREFIX + " access-key " + commandName + " <description>")
         .demand(/*count*/ 3, /*max*/ 3)  // Require exactly two non-option arguments.
-        .example("access-key " + commandName + " \"VSO Integration\"", "Generates a new access key with the description \"VSO Integration\"");
+        .example("access-key " + commandName + " \"VSO Integration\"", "Creates a new access key with the description \"VSO Integration\"");
 
     addCommonConfiguration(yargs);
 }
 
 function accessKeyList(commandName: string, yargs: yargs.Argv): void {
     isValidCommand = true;
-    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " [--format <format>]")
+    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " [options]")
         .demand(/*count*/ 2, /*max*/ 2)  // Require exactly two non-option arguments.
-        .example("access-key " + commandName, "Lists access keys in tabular format")
-        .example("access-key " + commandName + " --format json", "Lists apps in JSON format")
-        .option("format", { default: "table", demand: false, description: "The output format (\"json\" or \"table\")", type: "string" });
+        .example("access-key " + commandName, "Lists your access keys in tabular format")
```

Comment:
I'm going to leave it as "tabular", since that was the existing verbiage, and I think the term is common enough that I don't feel motivated to change it right now :)